keras_hub/src/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""LayoutLMv3 document classifier."""
```

Comment:
This file needs to be empty, all the import is handled in keras_hub/api directory and will be automatically generated  whenever you run `git commit -m "<message>"`
Make sure you run `pre-commit install` for the first time.